.github/workflows/comment-labeled.yml

```diff
@@ -22,8 +22,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: gh issue comment "$NUMBER" --repo ${{ github.repository }} --body "$STALE_MESSAGE"
 
-  fastTrack:
-    if: github.repository == 'nodejs/node' && github.event_name == 'pull_request_target' && github.event.label.name == 'fast-track'
+  fast-track:
+    if: github.repository == 'nodejs/node' && github.event.issue.pull_request && github.event.label.name == 'fast-track'
```

Comment:
I don't really understand this change, `github.event_name == 'pull_request_target'` is not equivalent to `!!github.event.issue.pull_request`, is it?

Reply:
`github.event.issue.pull_request` is `true` when the "issue" is a pull request, so yes it is.

Reply:
This unfortunately broke the `fast-track` label flow. See https://github.com/nodejs/node/pull/41888 and https://github.com/nodejs/node/actions/runs/1812810392 where it's skipped.